infrastructure/cli/cmd/create/helm.go

```diff
@@ -0,0 +1,221 @@
+	return h.runHelm([]string{"install", "--values", "/apps/config/airy-config-map.yaml", "core", "/apps/helm-chart/"})
+}
+
+func (h *Helm) runHelm(args []string) error {
+	if err := h.upsertAiryConfigMap(); err != nil {
+		return err
+	}
+
+	h.cleanupJob()
+	jobsClient := h.clientset.BatchV1().Jobs(h.namespace)
+
+	job := &batchv1.Job{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      h.name,
+			Namespace: h.namespace,
+			Labels:    map[string]string{"helm-runner": "true"},
+		},
+		Spec: batchv1.JobSpec{
+			Template: corev1.PodTemplateSpec{
+				Spec: corev1.PodSpec{
+					Containers: []corev1.Container{
+						{
+							Name:            "helm-runner",
+							Image:           "ghcr.io/airyhq/infrastructure/helm:" + h.version,
+							Args:            args,
+							ImagePullPolicy: corev1.PullAlways,
+							VolumeMounts: []corev1.VolumeMount{
+								{
+									Name:      "core-config",
+									MountPath: "/apps/config",
+								},
+							},
+						},
+					},
+					Volumes: []corev1.Volume{
+						{
+							Name: "core-config",
+							VolumeSource: corev1.VolumeSource{
+								ConfigMap: &corev1.ConfigMapVolumeSource{
+									LocalObjectReference: corev1.LocalObjectReference{
+										Name: airyConfigMap,
+									},
+								},
+							},
+						},
+					},
+					RestartPolicy:      "Never",
+					ServiceAccountName: serviceAccountName,
+				},
+			},
+		},
+	}
+
+	_, err := jobsClient.Create(context.TODO(), job, v1.CreateOptions{})
+	if err != nil {
+		return err
+	}
+
+	watcher, err := jobsClient.Watch(context.TODO(), v1.ListOptions{
+		LabelSelector: "helm-runner=true",
+	})
+	if err != nil {
+		return err
+	}
+
+	ch := watcher.ResultChan()
+
+	for event := range ch {
+		switch event.Type {
+		case watch.Error:
+			h.cleanupJob()
+			return fmt.Errorf("helm run failed with error %v", event.Object)
+		case watch.Added:
+		case watch.Modified:
+			job, _ := event.Object.(*batchv1.Job)
+			if job.Status.Succeeded == 1 {
+				h.cleanupJob()
+				return nil
+			} else if job.Status.Failed == 1 {
+				h.cleanupJob()
+				return fmt.Errorf("helm run failed with error %v", event.Object)
+			}
+		}
+	}
+
+	return nil
+}
+
+// Create/update airy config map
+func (h *Helm) upsertAiryConfigMap() error {
+	cm, _ := h.clientset.CoreV1().ConfigMaps(h.namespace).Get(context.TODO(), airyConfigMap, v1.GetOptions{})
+
+	cmData := map[string]string{
+		"airy-config-map.yaml": h.getAiryConfigMap(),
+	}
+
+	if cm.GetName() != "" {
+		cm.Data = cmData
+		_, err := h.clientset.CoreV1().ConfigMaps(h.namespace).Update(context.TODO(), cm, v1.UpdateOptions{})
+		return err
+	}
+
+	_, err := h.clientset.CoreV1().ConfigMaps(h.namespace).Create(context.TODO(),
+		&corev1.ConfigMap{
+			ObjectMeta: v1.ObjectMeta{
+				Name:      airyConfigMap,
+				Namespace: h.namespace,
+			},
+			Data: cmData,
+		}, v1.CreateOptions{})
+	return err
+}
+
+func (h *Helm) getAiryConfigMap() string {
+	content, err := ioutil.ReadFile("./infrastructure/defaults.yaml")
```

Comment:
I am not sure if this is going to work if you run the CLI from anywhere else than the repository. really think we need to embed this file for this to work.

Reply:
Can we come up with a test scenario for this? Or: how can we test this?

Reply:
You could build and copy the binary outside of the repository workspace and try running the `airy create` from there.